### DISCUSSION TITLE
I want to have two primary router

### DISCUSSION THREAD

**Discussion by NsdHSO** [vuejs/router#1892] [Help and Questions] [answered]
I saw Vue router support named router, But there is not what I want. For example, I have a list of carousels, and if I click on one carousel and the sidebar that will be displayed I want to be a child of the `default` router.
I don't know if it is possible with current features.

**Comment by posva** [comment, marked as answer]
this sounds like [Nested Routes](https://router.vuejs.org/guide/essentials/nested-routes.html#nested-routes)
